Documentation/operations/upgrade.rst

```diff
@@ -346,6 +346,11 @@ Removed Options
   ``disable-conntrack`` option and made it non-operational. It will be removed
   in version 1.13.
 
+Deprecated Options
+~~~~~~~~~~~~
```

Comment:
can you make underline here longer to make the tittle length ? Below make command can be used to verify

```
$ make -C Documentation
...
./operations/upgrade.rst:350: (WARNING/2) Title underline too short.
...
```

Reply:
Oh you beat me to it! :open_mouth: